grpc-proto/src/main/java/io/stargate/grpc/Values.java

```diff
@@ -163,4 +165,143 @@ public static Value of(Map<Value, Value> fields) {
         .setCollection(Collection.newBuilder().addAllElements(values).build())
         .build();
   }
+
+  public static boolean bool(Value value) {
+    if (value.getInnerCase() != InnerCase.BOOLEAN) {
+      throw new IllegalArgumentException("Expected boolean value");
+    }
+    return value.getBoolean();
+  }
+
+  public static int int_(Value value) {
+    if (value.getInnerCase() != InnerCase.INT) {
+      throw new IllegalArgumentException("Expected int value");
+    }
+    int intValue = (int) value.getInt();
+    if (intValue != value.getInt()) {
```

Comment:
Would this actually fail? As opposed to using same coercion here as in above line.
(I did add a note to test this case to make sure)

Reply:
it returns a `long`

Reply:
Ok then that makes sense (even if odd naming convention :) ).